core/src/main/java/org/elasticsearch/action/delete/DeleteResponse.java

```diff
@@ -94,7 +89,8 @@ public String toString() {
         builder.append(",type=").append(getType());
         builder.append(",id=").append(getId());
         builder.append(",version=").append(getVersion());
-        builder.append(",found=").append(found);
+        builder.append(",found=").append(isFound());
```

Comment:
I don't think you need to keep the `found` part of the `toString`. I'd just nuke this line.